tests/unittests/config/test_cc_package_update_upgrade_install.py

```diff
@@ -241,21 +241,16 @@ def test_error_apt(self, common_mocks):
         """
 
         def _new_subp(*args, **kwargs):
-            if args and "apt-cache" in args[0]:
-                return SubpResult("pkg1", None)
-            if "args" in kwargs and "install" in kwargs["args"]:
-                raise subp.ProcessExecutionError(
-                    cmd=kwargs["args"],
-                    stdout="dontcare",
-                    stderr="E: Unable to locate package pkg1",
-                    exit_code=100,
-                )
+            raise subp.ProcessExecutionError(
```

Comment:
While the end result is the same, this changes the call path and where we're raising exceptions. I want to get past the apt cache phase and fail particularly in the apt install path as that is known to raise exceptions. I'm open to better indicating that in this test, but I'm not sure why this change is being made here.